app/models/datafile.rb

```diff
@@ -34,7 +34,7 @@ def html?
   end
 
   def csv?
-    format&.upcase == 'CSV'
+    format[0..2]&.upcase == 'CSV'
```

Comment:
Is the trailing space going to be a problem for any of the other methods? 
What about trimming `format` in `initialize` or potentially override the accessor to return a trimmed version? I'm not really sure what this class is used for in the broad scheme of things though so I don't know if that would introduce any bugs somewhere else.

Reply:
Sure, I could trim it in initialize, I thought that just getting the first 3 chars would be more efficient but am open to changing it although at the same time, I know that only the first 3 chars are needed but this could potentially hide incorrect formats as well.